tests/basis_test.py

```diff
@@ -74,9 +45,9 @@ def test_cheb_call_random_points_multi_input():
     f0 = ChebyshevFirstKind(0)
     f1 = ChebyshevFirstKind(1)
     fn = ChebyshevFirstKind(5)
-    assert numpy.allclose(f0(xs),special.eval_chebyt(0,xs))
-    assert numpy.allclose(f1(xs),special.eval_chebyt(1,xs))
-    assert numpy.allclose(fn(xs),special.eval_chebyt(5,xs))
+    assert numpy.allclose(f0(xs), special.eval_chebyt(0, xs))
+    assert numpy.allclose(f1(xs), special.eval_chebyt(1, xs))
+    assert numpy.allclose(fn(xs), special.eval_chebyt(5, xs))
 
 
 def test_cheb_derivative():
```

Comment:
Please apply the comments about parametrizing the tests everywhere it is relevant in this file. You can pair n's with expected values, if you know them.